tree/ntuple/v7/test/ntuple_basics.cxx

```diff
@@ -758,3 +758,23 @@ TEST(REntry, Basics)
    e->BindRawPtr("pt", &pt);
    EXPECT_EQ(&pt, e->GetPtr<void>("pt").get());
 }
+
+TEST(RFieldBase, CreateObject)
+{
+   auto ptrInt = RField<int>("name").CreateObject<int>();
+   EXPECT_EQ(0, *ptrInt);
+
+   void *rawPtr = RField<int>("name").CreateObject<void>().release();
```

Comment:
The test should `delete` this raw pointer, otherwise Valgrind and some sanitizers will be angry at us...